scripts/west_commands/runners/nrfutil.py

```diff
@@ -77,7 +77,7 @@ def do_get_boards(self):
         for o in out:
             if o['type'] == 'task_end':
                 devs = o['data']['data']['devices']
-        snrs = [dev['serialNumber'] for dev in devs]
+        snrs = [dev['serialNumber'] for dev in devs if dev['traits']['devkit']]
```

Comment:
I think checking for the 'jlink' trait would make more sense here. The 'devkit' trait is only present for devkits from Nordic, so this check will limit the runner  to only work with these kits.

Checking for the 'jlink' trait will also work with stand-alone J-Link debuggers.